game.py

```diff
@@ -0,0 +1,13 @@
+
+class GameState:
+    def __init__(self):
+        self.state = "none"
+        self.step = 1
+        self.rng_seed_num = 160
+        self.start_time = 0
+
+# Global
+_game = GameState()
```

Comment:
I don't really like the idea of having critical global stuff around, but I also think that we can't really get away from it with how the TAS works. Not without a pretty significant overhaul of how things work. For the moment, this is. What I can say is that I don't think we need `get_gamestate`. I think we can just do:

```python
from game import _game
```

and use that outright. Might be worth calling it `game` instead of `_game` since it's not really private.

Reply:
I don't like the globals either, but something like it was needed once I put things into functions. One could argue for having it still be local only to main, but that would quickly fail if we decide to refactor out the perform_TAS() section to another file. Trying to think ahead.

That being said, I'll think about the suggestions above; I agree they make sense.

Reply:
Applied changes.